website/server/controllers/api-v3/auth.js

```diff
@@ -239,17 +241,39 @@ function _passportFbProfile (accessToken) {
   });
 }
 
+function _passportGoogleProfile (accessToken) {
+  return new Bluebird((resolve, reject) => {
```

Comment:
There's also a bluebird method to convert callbacks like this to promises automatically. That's probably what should actually be done for both of these.